src/fsharp/vs/Symbols.fs

```diff
@@ -1688,6 +1688,32 @@ and FSharpType(cenv, typ:TType) =
        protect <| fun () -> 
         "type " + NicePrint.stringOfTy (DisplayEnv.Empty(cenv.g)) typ 
 
+    member x.Symbol =
```

Comment:
I think it would be better to factor this as an "AsSymbol()" method on FSharpEntity.  Then you use fSharpType.TypeDefinition.AsSymbol().

Could you make that change?